apps/api/src/routes/signatureRouter.ts

```diff
@@ -170,41 +218,98 @@ signatureRouter.post(
  */
 signatureRouter.delete(
 	'/:applicationId',
+	authMiddleware(),
 	withParamsSchemaValidation(
 		deleteSignatureParamsSchema,
 		apiZodErrorMapping,
 		withQuerySchemaValidation(
 			deleteSignatureQuerySchema,
 			apiZodErrorMapping,
-			async (request: Request, response: Response) => {
-				const { applicationId } = request.params;
-				const { signee } = request.query;
+			async (
+				request: Request,
+				response: ResponseWithData<void, ['NOT_FOUND', 'UNAUTHORIZED', 'FORBIDDEN', 'SYSTEM_ERROR', 'INVALID_REQUEST']>,
+			) => {
+				try {
+					// Validate Parameter
+					const applicationId = Number(request.params.applicationId);
 
-				if (!applicationId || !signee || (signee !== 'APPLICANT' && signee !== 'INSTITUTIONAL_REP')) {
-					response.status(400).send({ message: 'Missing Required Parameters.' });
-					return;
-				}
+					if (!isPositiveInteger(applicationId)) {
+						response.status(400).send({ error: 'INVALID_REQUEST', message: 'Application ID is not a valid number.' });
+						return;
+					}
 
-				const result = await deleteApplicationSignature({
-					applicationId: Number(applicationId),
-					signee: signee,
-				});
+					// Validate Query Params
+					const queryValidationResult = deleteSignatureQuerySchema.safeParse(request.query);
+					if (!queryValidationResult.success) {
+						response.status(400).send({
+							error: 'INVALID_REQUEST',
+							message: `5Signee parameter must be either 'APPLICANT' or 'INSTITUTIONAL_REP'.`,
```

Comment:
Typo:
```suggestion
							message: `Signee parameter must be either 'APPLICANT' or 'INSTITUTIONAL_REP'.`,
```

However, regarding even having a parameter, I was hoping that, maybe as a future refactor this parameter would no longer be required. We can technically dynamically determine if they're a applicant or institutional rep through auth, so I feel like having this parameter feels like repetition.

Can DAC members delete parts of a document? if not, then we should be good to remove this in the future and just use Auth.

Reply:
Can be updated in future work. I was trying not to modify the interfaces here, so I didn't need to change the UI interactions with the API... that would make this PR much much too big, rather than just too big.